bin/configure.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Configure script used by fluo-dev & fluo-deploy
+
+BIN_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+PC_HOME=$( cd "$( dirname "$BIN_DIR" )" && pwd )
+
+if [ "$#" -ne 1 ]; then
```

Comment:
I am undecided on this comment.

Could drop this check as the next, since TXT_DIR is not used in this script.    Or the checks could be left as a sanity check.  I like the idea of having the sanity check, but I also like the idea of making the script shorter.